openapi.yaml

```diff
@@ -4981,6 +5183,19 @@ components:
                       $ref: '#/components/schemas/Error401RequestAuth'
     Error402:
       description: Payment Required.
+      headers:
```

Comment:
Vypadá to OK, trochu mě překvapilo že credit hlavičky se vrací i v případě erorrů, trochu to vypadá jako kdyby API mohlo spotřebovávat kredity i když pak vrátí error.

Reply:
Hm, pravda že se tam ty credit hlavičky neposílají, Dominik tvrdil, že se posílají hlavičky všechny, tak to asi přehlédl. Opravím :wink: